src/main/java/com/dumptruckman/bukkit/configuration/util/SerializationHelper.java

```diff
@@ -124,25 +130,32 @@ public static Object deserialize(@NotNull final Map<?, ?> input) {
             try {
                 return ConfigurationSerialization.deserializeObject(output);
             } catch (IllegalArgumentException ex) {
-                throw new YAMLException("Could not deserialize object", ex);
+                LOG.severe("Could not deserialize the following object:");
+                LOG.severe(String.valueOf(output));
+                if (!continueOnSerializationError) {
+                    throw new SerializerException(ex.getMessage());
+                }
+                return null;
```

Comment:
I think it should only log if we're not going to throw.

Reply:
Previously it log and throw as well, so i didnt change that behaviour, also i think its good to print out the particluar object that has error, its easier to debug.